tests/regtest.rs

```diff
@@ -107,6 +107,18 @@ fn prepare_btc_claim() -> (
     )
 }
 
+#[test]
+fn btc_reverse_claim_size() {
+    let (_test_framework, _scan_request, swap_tx, _preimage, recvr_keypair, _utxos) =
+        prepare_btc_claim();
+
+    let coop_claim_tx_size = swap_tx.size(&recvr_keypair, true).unwrap();
+    assert_eq!(coop_claim_tx_size, 84);
```

Comment:
84 bytes for a cooperative claim transaction? That does not look right. Could you double check that? Are you adding an output?